adm/_metwork.spec

```diff
@@ -302,6 +302,19 @@ chmod -R a+rX %{buildroot}{{MFMODULE_HOME}}
 rm -Rf %{_builddir}/%{name}-%{version}-{{RELEASE_BUILD}} 2>/dev/null
 
 
+{% if MFEXT_ADDON == "0" %}
+############################################################
+##### post SECTION (POSTINSTALLATION) FOR MAIN PACKAGE #####
+############################################################
+%post
+    {% if MFMODULE != "MFEXT" %}
+        if test -f /home/.home_{{MFMODULE_LOWERCASE}}.perm; then
```

Comment:
remove indentation here

Reply:
??

Reply:
the "{% if" it's jinja2 templating so once the templating will be "translated" it will result in unnecessary indentations in the spec file, but OK you can keep it for developer's readability